node/src/actors/session/handlers.rs

```diff
@@ -793,6 +844,12 @@ fn send_inventory_item_msg(session: &mut Session, item: InventoryItem) {
             // Send Transaction msg
             session.send_message(transaction_msg);
         }
+        InventoryItem::SuperBlock(superblock) => {
+            // Build Transaction msg
```

Comment:
Copy-pasted comments

```suggestion
            // Build Superblock msg
```